sdk/storage/Azure.Storage.Common/src/Shared/StorageSharedKeyCredentialExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage
+{
+    internal class StorageSharedKeyCredentialExtensions : StorageSharedKeyCredential
+    {
+        public StorageSharedKeyCredentialExtensions(string accountName, string accountKey) :
```

Comment:
Why bother with a public .ctor?  Can we just have a `private StorageSharedKeyCredentialExtensions() { }` so nobody can instantiate?

Reply:
We need a constructor that supplies the required parameters of StorageSharedKeyCredential. I can make it private.